### ISSUE TITLE
check-rpc-mappings.py is failing on name_filter

### ISSUE THREAD

**Issue by JeremyRand** [namecoin/namecoin-core#203]
The `check-rpc-mappings.py` step on Travis is failing:

~~~
$ if [ "$CHECK_DOC" = 1 ]; then contrib/devtools/check-rpc-mappings.py .; fi
* Checking consistency between dispatch tables and vRPCConvertParams
Traceback (most recent call last):
  File "contrib/devtools/check-rpc-mappings.py", line 158, in <module>
    main()
  File "contrib/devtools/check-rpc-mappings.py", line 114, in main
    rargnames = cmds_by_name[cmdname].args[argidx].names
KeyError: 'name_filter'
The command "if [ "$CHECK_DOC" = 1 ]; then contrib/devtools/check-rpc-mappings.py .; fi" failed and exited with 1 during .
~~~

Full log at https://travis-ci.org/namecoin/namecoin-core/jobs/328450893#L664

**Comment by JeremyRand** [comment]
@domob1812 Any idea what's wrong?

**Comment by domob1812** [comment]
Good catch.  Fix is in #207.
